web/src/components/organisms/atoms/Navbar.jsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+import { HiLogout } from "react-icons/hi";
+
+const Navbar = () => {
+    return (
+<nav className="bg-gray-50 dark:bg-gray-800 p-2 text-center">
+      <div className="container mx-auto flex justify-between items-center">
```

Comment:
- Remove unnecessary class
- Lets add px-4 so each side can have similar spaces

```suggestion
    <nav className='bg-gray-50 dark:bg-gray-800 p-2'>
      <div className='px-4 flex justify-between items-center'>
```